### ISSUE TITLE
[Accessibility, a11y] Screen reader isn't reading quick pick placeholder field

### ISSUE THREAD

**Issue by MicroFish91** [microsoft/vscode#195723]
**VS Code Version**:  1.84.0-insider

**Operating System**:  macOS v14.0

**Screen Reader**:  Apple VoiceOver

**Extensions Installed**:
Put together an extension to quickly demo the bare VS Code UI features without our normal SDK and extension wrappers.
"a11y-kit" (https://marketplace.visualstudio.com/items?itemName=MicroFish.accessibility-testkit)

**Repro Steps**:
  1. Install a11y-kit from VS Code Marketplace
  2. Select: `a11y-kit: Select UI feature for accessibility testing`
  3. Select: `Show Quick Picks - Single` and observe.

**Expected**:
The `Quick Pick Placeholder` text should be read.
![image](https://github.com/microsoft/vscode/assets/40250218/2912c8e6-036b-4a66-bcb4-388fa4507da5)

**Actual**:
The placeholder text is not read.

**Originating reference issue**:
https://github.com/microsoft/vscode-azurefunctions/issues/3820

**Comment by TylerLeonhardt** [comment]
Does this work with NVDA?

**Comment by MicroFish91** [comment]
It seems to be working with NVDA

**Comment by TylerLeonhardt** [comment]
Ok so it works with NVDA but not VoiceOver... this is likely VoiceOver not handling `aria-activedescendant` correctly.

**Comment by meganrogge** [comment]
I can reproduce on mac, I suggest using aria alert here

**Comment by TylerLeonhardt** [comment]
I'd rather we push on VoiceOver to make a change than introduce an aria-alert hack.

**Comment by TylerLeonhardt** [comment]
Here's a codepen of the issue.

https://codepen.io/tylerleonhardt/pen/dyaMjgz

Forked from https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-none/

CHANGES MADE:
* Deleted label and replaced with a placeholder attribute on the input
* Added `aria-activedescendant="lb1-01"` to input
* Commented out JS code that sets `aria-activedescendant` to force a Quick Pick behavior

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by kupatkar99** [comment]
Verified the issue on similar type of control:
Visual studio Code Version: 1.82.1 (Universal)
Issue is fixed. hence closing the bug please find added attcahment for reference.
MicrosoftTeams-image

**Comment by TylerLeonhardt** [comment]
Closing due to @kupatkar99's update.
